apps/molgenis-components/src/components/forms/RowEdit.vue

```diff
@@ -27,9 +27,7 @@
   </div>
 </template>
 
-<script lang="ts">
```

Comment:
Had to remove ts because it kept telling me the default export had implicit any type, and I couldn't figure out how to fix it :(